src/Search/Microsoft.Azure.Search/GeneratedSearchService/Models/StandardTokenizer.cs

```diff
@@ -39,8 +39,10 @@ public StandardTokenizer(string name, int? maxTokenLength = default(int?))
         }
 
         /// <summary>
-        /// Gets or sets the maximum token length. Tokens longer than the
-        /// maximum length are split. Default is 255.
+        /// Gets or sets the maximum token length. Default is 255. Tokens
+        /// longer than the maximum length are split. The maximum token
+        /// length that can be used is 300 characters. Tokens longer than 300
+        /// characters are ignored.
```

Comment:
This is not true for this tokenizer, just checked. We will not allow setting the limit to more than 300, as you wrote, and tokens longer than the limit will be split.